Sunrise/ViewModels/OrdersViewModel.swift

```diff
@@ -0,0 +1,126 @@
+//
+// Copyright (c) 2016 Commercetools. All rights reserved.
+//
+
+import ReactiveCocoa
+import Result
+import ObjectMapper
+import Commercetools
+
+class OrdersViewModel: BaseViewModel {
+
+    // Inputs
+    let refreshObserver: Observer<Void, NoError>
+    let sectionExpandedObserver: Observer<Int, NoError>
+
+    // Outputs
+    let isLoading: MutableProperty<Bool>
+    let contentChangesSignal: Signal<Changeset, NoError>
+    let ordersExpanded = MutableProperty(false)
+    let reservationsExpanded = MutableProperty(false)
+
+    var orders = [Order]()
+    var reservations = [Order]()
+
+    private let contentChangesObserver: Observer<Changeset, NoError>
+
+    // MARK: - Lifecycle
+
+    override init() {
+        isLoading = MutableProperty(true)
+
+        let (refreshSignal, observer) = Signal<Void, NoError>.pipe()
+        refreshObserver = observer
+
+        let (sectionExpandedSignal, expandedObserver) = Signal<Int, NoError>.pipe()
+        sectionExpandedObserver = expandedObserver
+
+        let (signal, changesObserver) = Signal<Changeset, NoError>.pipe()
+        contentChangesSignal = signal
+        contentChangesObserver = changesObserver
+
+        super.init()
+
+        refreshSignal
+        .observeNext { [weak self] in
```

Comment:
@lauraluiz @cneijenhuis - do you think we should have pull-to-refresh on the my account screen as well? It might be useful for demos, when an order is created from the web app?